<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>687</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>500</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="MySFMLCanvas" name="MyCanvas" native="true"/>
    </item>
    <item>
     <widget class="QGroupBox" name="InfoBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Info</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="circuit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>150</height>
          </size>
         </property>
         <property name="title">
          <string>Circuit</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="circuitName">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="circuitName_2">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="circuitDieArea">
            <property name="text">
             <string>Die Area</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="circuitDieArea_2">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="circuitCells">
            <property name="text">
             <string>Cells</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="circuitCells_2">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="circuitPins">
            <property name="text">
             <string>Pins</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="circuitPins_2">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="circuitNets">
            <property name="text">
             <string>Nets</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="circuitNets_2">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="selectedCell">
         <property name="title">
          <string>Selected Cell</string>
         </property>
         <widget class="QWidget" name="NameType" native="true">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>20</y>
            <width>51</width>
            <height>61</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="selectedCellName">
             <property name="text">
              <string>Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="selectedCellType">
             <property name="text">
              <string>Type</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="widget_2" native="true">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>40</y>
            <width>120</width>
            <height>80</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="selectedCellWSlack">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>170</y>
            <width>71</width>
            <height>19</height>
           </rect>
          </property>
          <property name="text">
           <string>Worst Slack</string>
          </property>
         </widget>
         <widget class="QLabel" name="selectedCellPosition">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>80</y>
            <width>51</width>
            <height>27</height>
           </rect>
          </property>
          <property name="text">
           <string>Position</string>
          </property>
         </widget>
         <widget class="QWidget" name="NameType_2" native="true">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>20</y>
            <width>171</width>
            <height>61</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="selectedCellType_2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="selectedCellName_2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QLabel" name="selectedCell_y">
          <property name="geometry">
           <rect>
            <x>62</x>
            <y>108</y>
            <width>16</width>
            <height>26</height>
           </rect>
          </property>
          <property name="text">
           <string>y:</string>
          </property>
         </widget>
         <widget class="QLabel" name="selectedCell_x">
          <property name="geometry">
           <rect>
            <x>62</x>
            <y>76</y>
            <width>16</width>
            <height>35</height>
           </rect>
          </property>
          <property name="text">
           <string>x:</string>
          </property>
         </widget>
         <widget class="QLabel" name="selectedCellWSlack_2">
          <property name="geometry">
           <rect>
            <x>88</x>
            <y>170</y>
            <width>131</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
         <widget class="QDoubleSpinBox" name="selectedCell_y_2">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>111</y>
            <width>136</width>
            <height>26</height>
           </rect>
          </property>
          <property name="minimum">
           <double>-10000000000000000564754110205208414148406263819830583747005651641554565639675781971892197615894599829797681693475363620965659806446069238773051601456032797794197839403040623198185642380825912769195995883053017532724018486962951290880.000000000000000</double>
          </property>
          <property name="maximum">
           <double>10000000000000000564754110205208414148406263819830583747005651641554565639675781971892197615894599829797681693475363620965659806446069238773051601456032797794197839403040623198185642380825912769195995883053017532724018486962951290880.000000000000000</double>
          </property>
         </widget>
         <widget class="QDoubleSpinBox" name="selectedCell_x_2">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>81</y>
            <width>136</width>
            <height>26</height>
           </rect>
          </property>
          <property name="minimum">
           <double>-10000000000000000564754110205208414148406263819830583747005651641554565639675781971892197615894599829797681693475363620965659806446069238773051601456032797794197839403040623198185642380825912769195995883053017532724018486962951290880.000000000000000</double>
          </property>
          <property name="maximum">
           <double>10000000000000000564754110205208414148406263819830583747005651641554565639675781971892197615894599829797681693475363620965659806446069238773051601456032797794197839403040623198185642380825912769195995883053017532724018486962951290880.000000000000000</double>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>79</x>
            <y>141</y>
            <width>135</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Update Position</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="netlist">
         <property name="title">
          <string>Netlist</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2"/>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen_LEF_DEF">
     <property name="title">
      <string>Open LEF/DEF</string>
     </property>
     <addaction name="actionICCAD_2017"/>
     <addaction name="actionICCAD_2015"/>
    </widget>
    <addaction name="menuOpen_LEF_DEF"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionICCAD_2017">
   <property name="text">
    <string>ICCAD 2017</string>
   </property>
  </action>
  <action name="actionICCAD_2015">
   <property name="text">
    <string>ICCAD 2015</string>
   </property>
  </action>
  <action name="actionOpen_placement_solution_DEF">
   <property name="text">
    <string>Open placement solution (.DEF)</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MySFMLCanvas</class>
   <extends>QWidget</extends>
   <header>view/mysfmlcanvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
